src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/dialog/preferences/PreferencesPages.kt

```diff
@@ -394,6 +395,164 @@ object PreferencesPages {
+                    )
+                    float(
+                        title = Strings.PreferencesChartsFundamentalErbsStep,
+                        defaultValue = AppConf.Fundamental.DEFAULT_ERBS_STEP,
+                        min = AppConf.Fundamental.MIN_ERBS_STEP,
+                        max = AppConf.Fundamental.MAX_ERBS_STEP,
+                        select = { it.erbsStep },
+                        update = { copy(erbsStep = it) },
+                    )
+                    float(
+                        title = Strings.PreferencesChartsFundamentalMinDisplayCorr,
+                        defaultValue = AppConf.Fundamental.DEFAULT_MIN_DISPLAY_CORR,
+                        min = 0.0f,
+                        max = 1.0f,
+                        select = { it.minDisplayCorr },
+                        update = { copy(minDisplayCorr = it) },
+                        validationRules = listOf(
+                            PreferencesItemValidationRule(
+                                validate = { appConf ->
+                                    appConf.painter.fundamental.let { it.minDisplayCorr < it.maxDisplayCorr }
+                                },
+                                prompt = Strings.PreferencesChartsFundamentalMinDisplayCorrInvalid,
+                            ),
+                        ),
+                    )
+                    float(
+                        title = Strings.PreferencesChartsFundamentalMaxDisplayCorr,
+                        defaultValue = AppConf.Fundamental.DEFAULT_MAX_DISPLAY_CORR,
+                        min = 0.0f,
+                        max = 1.0f,
+                        select = { it.maxDisplayCorr },
+                        update = { copy(maxDisplayCorr = it) },
+                        validationRules = listOf(
+                            PreferencesItemValidationRule(
+                                validate = { appConf ->
+                                    appConf.painter.fundamental.let { it.minDisplayCorr < it.maxDisplayCorr }
+                                },
+                                prompt = Strings.PreferencesChartsFundamentalMaxDisplayCorrInvalid,
+                            ),
+                        ),
+                    )
+                    switch(
+                        title = Strings.PreferencesChartsFundamentalDrawReferenceLine,
+                        defaultValue = AppConf.Fundamental.DEFAULT_DRAW_REFERENCE_LINE,
+                        select = { it.drawReferenceLine },
+                        update = { copy(drawReferenceLine = it) },
+                    )
+                    color(
+                        title = Strings.PreferencesChartsFundamentalColor,
+                        defaultValue = AppConf.Fundamental.DEFAULT_COLOR,
+                        select = { it.color },
+                        update = { copy(color = it) },
+                        useAlpha = true,
+                    )
+                    color(
```

Comment:
We can move the reference line color next to the `draw reference line` and add `enabled` to it?

Reply:
I think the `color` should be shown at first, so I move the `draw reference line`.